serverless/aws/src/index.ts

```diff
@@ -206,6 +207,9 @@ export const handlerRaw = async (
         data = tilePostprocess(data, header.tileType);
       }
 
+      headers["Cache-Control"] = `public, max-age=${process.env.CACHE_MAX_AGE || 86400}`;
+      headers["eTag"] = `"${createHash("sha256").update(Buffer.from(data)).digest("hex")}"`
```

Comment:
even though headers are treated as case-insensitive I think the well-known case is `ETag` right?

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/ETag

Reply:
Good spotting, have updated it now.